apps/desktop/src/styles.css

```
@@ -21,36 +26,44 @@
+  --shadow-bleed-island: 96px;
+  --window-glass-tint: light-dark(rgba(255, 255, 255, 0.28), rgba(0, 0, 0, 0.11));
+  --window-noise-opacity: 0.12;
+  --window-noise-size: 512px;
   --native-titlebar-height: 38px;
-  --native-traffic-light-cluster-width: 74px;
+  --native-traffic-light-cluster-width: 60px;
   --native-home-link-left-default: var(--space-4);
   --native-home-link-left-macos: calc(
     var(--native-traffic-light-cluster-width) + var(--native-home-link-left-default)
   );
   color: var(--color-on-island);
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)